receiver/awsxrayreceiver/internal/translator/sql.go

```diff
@@ -0,0 +1,65 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"fmt"
+	"regexp"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/translator/conventions"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver/internal/tracesegment"
+)
+
+func addSQLToSpan(sql *tracesegment.SQLData, attrs *pdata.AttributeMap) error {
+	if sql == nil {
+		return nil
+	}
+
+	// https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/exporter/awsxrayexporter/translator/sql.go#L33
```

Comment:
Less important than having code and attributes in a shared location, so this can come later, but the translation in the common location might be nice too eventually, a class with `fromPdata` and `toPdata` in the same location is very easy to reason about instead of jumping around.

Reply:
Added a TODO